### ISSUE TITLE
[Feature][BliKVM v4] display both local LAN and VPN IP in OLED display

### ISSUE THREAD

**Issue by alandjbaker** [blikvm/blikvm#157]
It would be useful if on the oled display it showed both the network ip of zero tier and the local lan. 
There is space at the bottom of the oled display for a second IP.

**Comment by m50S79sM6SRNp8Jn** [comment]
For vue3 
npm install @casl/vue @casl/ability

I am starting to play with this.

**Comment by m50S79sM6SRNp8Jn** [comment]
Which BliKVM device are you using?

**Comment by alandjbaker** [comment]
> Which BliKVM device are you using?

The Allwinner V4

**Comment by m50S79sM6SRNp8Jn** [comment]
Thanks

**Comment by alandjbaker** [comment]
My BLIKVM is on its way three thousand miles away so I can support remote users so it would be useful for the end user to see both IP's (this would also indicate the KVM is online).

**Comment by m50S79sM6SRNp8Jn** [comment]
Nice, thanks for providing the rational.

**Comment by m50S79sM6SRNp8Jn** [comment]
Do you by any chance have other BliKVM's (which, quanity)?

**Comment by alandjbaker** [comment]
> Do you by any chance have other BliKVM's (which, quanity)?

Just the one I'm afraid, bought from AliExpress, delivered to the UK, configured it with zerotier and dispatched to UAE!

**Comment by m50S79sM6SRNp8Jn** [comment]
> It would be useful if on the oled display it showed both the network ip of zero tier and the local lan. There is space at the bottom of the oled display for a second IP.

Would it more or less useful to you having both the local LAN and VPN IP in the GUI, than in the BliKVM v4 Allwinner display?
Maybe we do both, so in the GUI and in the display, just want to get your feedback first please.

**Comment by m50S79sM6SRNp8Jn** [comment]
> In my use case it would be no use in the GUI as if I'm in the GUI then I've successfully connected to it.

o/c that makes sense, we'll do both (display + GUI)

**Comment by alandjbaker** [comment]
Yeah, to have this information in the GUI is surely going to be useful to some.

**Comment by m50S79sM6SRNp8Jn** [comment]
Feature added. Update to latest BliKVM
